src/base/bittorrent/peerinfo.cpp

```diff
@@ -43,7 +43,7 @@ PeerAddress::PeerAddress()
 {
 }
 
-PeerAddress::PeerAddress(QHostAddress ip, ushort port)
+PeerAddress::PeerAddress(const QHostAddress &ip, ushort port)
```

Comment:
Adding const and turning variables to const reference are quite different, you should mention this in the commit message.

Reply:
ok